TodoApp.ConsoleApp/UI/UserComments.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace TodoApp.ConsoleApp.UI
+{
+    static class UserComments
```

Comment:
Rename the class. `UseComments` sounds like we are commenting the user or he is giving some comments.
Examples: `Messages`, `Output`, `Text`, etc.